components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -118,6 +119,17 @@ spec:
                 name: {{ include "tyk-pump.pg_connection_string_secret_name" . }}
                 key: {{ include "tyk-pump.pg_connection_string_secret_key" . }}
           {{ end }}
+          {{ if has "postgres-pump" .Values.pump.backend }}
```

Comment:
Can we club this condition and ` {{ if has "postgres" .Values.pump.backend }}`?
They are doing the same thing.

Reply:
updated in 5962d8348cbc956bde0f1c62067fedcafd2633ce, can you please check again?